src/views/wallet/Transfer.vue

```diff
@@ -319,23 +365,147 @@ export default class Transfer extends Vue {
         }
     }
 
-    onerror(err: any) {
+    onError(err: any) {
         this.err = err
         this.isAjax = false
-        let { dispatchNotification } = this.globalHelper()
-        dispatchNotification({
+        this.helpers.dispatchNotification({
             message: this.$t('notifications.transfer_error_msg'),
             type: 'error',
         })
     }
+    @Watch('formType', { immediate: true })
+    updateDetails() {
+        if (this.formType === 'P') this.updateMultisigTxDetails()
+        else {
+            this.memo = ''
+            this.addressIn = ''
+        }
+    }
+    refresh() {
+        this.$store.dispatch('Signavault/updateTransaction')
+        this.updateMultisigTxDetails()
+    }
+    get pendingSendMultisigTX(): SignavaultTx | undefined {
+        return this.$store.getters['Signavault/transactions'].find(
+            (item: any) =>
+                item?.tx?.alias === this.wallet.getAllAddressesP()[0] &&
+                WalletHelper.getUnsignedTxType(item?.tx?.unsignedTx) === 'BaseTx'
+        )
+    }
+
+    get txOwners() {
+        return this.pendingSendMultisigTX?.tx?.owners ?? []
+    }
+
+    get activeWallet(): MultisigWallet {
+        return this.$store.state.activeWallet
+    }
+
+    async signMultisigTx() {
+        const wallet = this.activeWallet
+        if (!wallet || !(wallet instanceof MultisigWallet))
+            return console.debug('MultiSigTx::sign: Invalid wallet')
```

Comment:
Why cancel and only log on debug level?
This will then display properly to the user that something went wrong?